tests/model_serving/model_server/utils.py

```diff
@@ -0,0 +1,61 @@
+from contextlib import contextmanager
+from typing import Any, Dict, Optional
+
+from kubernetes.dynamic import DynamicClient
+from ocp_resources.inference_service import InferenceService
+
+
+@contextmanager
+def create_isvc(
+    client: DynamicClient,
+    name: str,
+    namespace: str,
+    deployment_mode: str,
+    storage_uri: str,
+    model_format: str,
+    runtime: str,
+    wait: bool = True,
+    enable_auth: bool = False,
+    external_route: bool = False,
+    model_service_account: Optional[str] = "",
+    min_replicas: Optional[int] = None,
+) -> InferenceService:
+    predictor_config: Dict[str, Any] = {
+        "model": {
+            "modelFormat": {"name": model_format},
+            "version": "1",
+            "runtime": runtime,
+            "storageUri": storage_uri,
```

Comment:
If you want to redefine this before #22 , please also make sure to include support for storage_key and storage_path, see: https://github.com/opendatahub-io/opendatahub-tests/pull/22/files/38b1f728c8d82955805212d36cf8c47f03d17a10#diff-afd99f559fc337b6071cc32a05efb2c1ad55e61e9d2bbc685545fc26a64cdd0fR17

Reply:
addressed in #22